docs/development/design/network-router.md

```diff
@@ -0,0 +1,93 @@
+# Network Router
+
+> Additional details and running code in [PR #981](https://github.com/nexodus-io/nexodus/pull/981)
+
+## Summary
+
+Add the ability for device's running nexd to access non-nexd devices and act as a router for external networks.
+
+## Proposal
+
+There are two supported use cases.
+
+1. Traffic from the Nexd mesh can reach external non-nexd nodes via source NAT without any configuration required.
+2. Traffic from the Nexd mesh can reach external non-nexd nodes but require a configuration for return traffic since SNAT is disabled. This scenario will also allow external devices to initiate traffic to the mesh since the devices are required to have a route to an interface on the device connected to the physical network. With SNAT being disabled, the non-nexd device can also initiate traffic from outside the mesh going in.
```

Comment:
Might be better to make #2 two different use cases even if they get implemented using the same strategy.

Reply:
You suggesting a separate doc or just putting it in a different section for NAT being disabled? Ty.

Reply:
Just to add a third bullet to this doc.